src/struct.jl

```diff
@@ -0,0 +1,38 @@
+using MixedModels
+using ForwardDiff
+using JSON3
+
+@kwdef struct KenwardRogerMatrices
+    fit::MixedModel
+    σ2s::Vector{Float64}
+    V::Array{Float64,2}
+    W::Array{Float64,2}
+    P::Vector{Matrix{Float64}}
+    Q::Matrix{Matrix{Float64}}
+    CovVar::Array{Float64,2}
+    CovBeta::Array{Float64,2}
+    StdBetas::Array{Float64,1}
+end
+
+
+@kwdef struct CovMatrixVarComponents
+    ids::Vector{String}
+    values::Vector{String}
+end
+
+@kwdef struct FixedEffect
```

Comment:
We should return an object which fits in the already existing type hierarchy https://github.com/JuliaStats/MixedModels.jl/blob/f7b201700221c07e45d4de2fd7d99db1a974939c/src/MixedModels.jl#L178